other/bsp/riscv64/licheerv-nano/bootloader

```diff
@@ -5,12 +5,15 @@ function internal_build_bootloader {
 
    echo "Building bootloader... "
    reset_cc_vars_to_null
+   export PATH=${PATH//$GCC_TOOLCHAIN:/}
```

Comment:
We already have a safer pattern for this case, see `build_libmusl`.
In practice, you need to save the path using a local variable:
```
local saved_path="$PATH"
```
Then you modify it as you want.
At the end of the function, you restore the path with:
```
export PATH="$saved_path"
```
Can you please use that pattern?